seerep-ros/seerep_ros_conversions_fb/test/ros_to_fb_conversion_test.cpp

```diff
@@ -0,0 +1,673 @@
+
+    flatbuffers::grpc::Message<seerep::fb::Vector3Stamped> fb_vector3_stamped;
+    fb_vector3_stamped = seerep_ros_conversions_fb::toFlat(original_v3_stamped, p_uuid);
+
+    geometry_msgs::Vector3Stamped converted_v3_stamped =
+        seerep_ros_conversions_fb::toROS(*fb_vector3_stamped.GetRoot());
+    // Vector3Stamped End
+
+    // Transform Start
+    geometry_msgs::Transform original_t = createTransform();
+
+    flatbuffers::grpc::Message<seerep::fb::Transform> fb_transform;
+    fb_transform = seerep_ros_conversions_fb::toFlat(original_t);
+
+    geometry_msgs::Transform converted_t = seerep_ros_conversions_fb::toROS(*fb_transform.GetRoot());
+    // Transform End
+
+    // TransformStamped Start
+    geometry_msgs::TransformStamped original_t_stamped = createTransformStamped();
+
+    flatbuffers::grpc::Message<seerep::fb::TransformStamped> fb_transform_stamped;
+    fb_transform_stamped = seerep_ros_conversions_fb::toFlat(original_t_stamped, p_uuid);
+
+    geometry_msgs::TransformStamped converted_t_stamped =
+        seerep_ros_conversions_fb::toROS(*fb_transform_stamped.GetRoot());
+    // Transform End
+  }
+};
+
+// Objects created below are used by the TEST functions
+
+// Header
+std_msgs::Header rosToFbConversionTest::original_header;
+std_msgs::Header rosToFbConversionTest::converted_header;
+
+// PointField
+sensor_msgs::PointField rosToFbConversionTest::original_pf;
+sensor_msgs::PointField rosToFbConversionTest::converted_pf;
+
+// PointCloud2
+sensor_msgs::PointCloud2 rosToFbConversionTest::original_pc2;
+sensor_msgs::PointCloud2 rosToFbConversionTest::converted_pc2;
+
+// Image
+sensor_msgs::Image rosToFbConversionTest::original_img;
+sensor_msgs::Image rosToFbConversionTest::converted_img;
+
+// Point
+geometry_msgs::Point rosToFbConversionTest::original_p;
+geometry_msgs::Point rosToFbConversionTest::converted_p;
+
+// Quaternion
+geometry_msgs::Quaternion rosToFbConversionTest::original_q;
+geometry_msgs::Quaternion rosToFbConversionTest::converted_q;
+
+// Pose
+geometry_msgs::Pose rosToFbConversionTest::original_pose;
+geometry_msgs::Pose rosToFbConversionTest::converted_pose;
+
+// PoseStamped
+geometry_msgs::PoseStamped rosToFbConversionTest::original_pose_stamped;
+geometry_msgs::PoseStamped rosToFbConversionTest::converted_pose_stamped;
+
+// Vector3
+geometry_msgs::Vector3 rosToFbConversionTest::original_v;
+geometry_msgs::Vector3 rosToFbConversionTest::converted_v;
+
+// Vector3Stamped
+geometry_msgs::Vector3Stamped rosToFbConversionTest::original_v3_stamped;
+geometry_msgs::Vector3Stamped rosToFbConversionTest::converted_v3_stamped;
+
+// Transform
+geometry_msgs::Transform rosToFbConversionTest::original_t;
+geometry_msgs::Transform rosToFbConversionTest::converted_t;
+
+// TransformStamped
+geometry_msgs::TransformStamped rosToFbConversionTest::original_t_stamped;
+geometry_msgs::TransformStamped rosToFbConversionTest::converted_t_stamped;
```

Comment:
Is this really necessary? Isn't it easier to create an object of the class?
Would the structure of the test benefit from putting everything in the class?